DOCUMENTATION.md

```diff
@@ -447,54 +614,113 @@ type Service internal (resolve : ClientId -> Equinox.Stream<Events.Event, Fold.S
+  _Read Model_
 
 ## Programming Model walkthrough
 
 ### Flows and Streams
 
-Equinox’s Command Handling consists of < 200 lines including interfaces and comments in https://github.com/jet/equinox/tree/master/src/Equinox - the elements you'll touch in a normal application are:
-
-- [`module Flow`](https://github.com/jet/equinox/blob/master/src/Equinox/Flow.fs#L34) - internal implementation of Optimistic Concurrency Control / retry loop used by `Stream`. It's recommended to at least scan this file as it defines the Transaction semantics everything is coming together in service of.
-- [`type Stream`](https://github.com/jet/equinox/blob/master/src/Equinox/Equinox.fs#L11) - surface API one uses to `Transact` or `Query` against a specific stream
-- [`type Target` Discriminated Union](https://github.com/jet/equinox/blob/master/src/Equinox/Equinox.fs#L42) - used to identify the Stream pertaining to the relevant Aggregate that `resolve` will use to hydrate a `Stream`
-
-Its recommended to read the examples in conjunction with perusing the code in order to see the relatively simple implementations that underlie the abstractions; the 2 files can tell many of the thousands of words about to follow!
+Equinox’s Command Handling consists of < 200 lines including interfaces and
+comments in https://github.com/jet/equinox/tree/master/src/Equinox - the
+elements you'll touch in a normal application are:
+
+- [`module Flow`](https://github.com/jet/equinox/blob/master/src/Equinox/Flow.fs#L34) -
+  internal implementation of Optimistic Concurrency Control / retry loop used
+  by `Stream`. It's recommended to at least scan this file as it defines the
+  Transaction semantics everything is coming together in service of.
+- [`type Stream`](https://github.com/jet/equinox/blob/master/src/Equinox/Equinox.fs#L11) -
+  surface API one uses to `Transact` or `Query` against a specific stream
+- [`type Target` Discriminated Union](https://github.com/jet/equinox/blob/master/src/Equinox/Equinox.fs#L42) -
+  used to identify the Stream pertaining to the relevant Aggregate that
+  `resolve` will use to hydrate a `Stream`
+
+Its recommended to read the examples in conjunction with perusing the code in
+order to see the relatively simple implementations that underlie the
+abstractions; the 2 files can tell many of the thousands of words about to
+follow!
 
 #### Stream Members
 
 ```fsharp
 type Equinox.Stream(stream : IStream<'event, 'state>, log, maxAttempts) =
 StoreIntegration
-    // Run interpret function with present state, retrying with Optimistic Concurrency
+    // Run interpret function with present state, retrying with Optimistic
+    // Concurrency
     member __.Transact(interpret : State -> Event list) : Async<unit>
 
-    // Run decide function with present state, retrying with Optimistic Concurrency, yielding Result on exit
+    // Run decide function with present state, retrying with Optimistic
```

Comment:
again, I trued to make this a pithy one line, which this destroys the spirit of

Reply:
Can `\n` go after `,` or do you want the whole thing back to original?

Reply:
that is, `Concurrency, yielding ` --> `Concurrency,\nyielding`

Reply:
open to splitting on comma